client/jetpack-cloud/sections/agency-dashboard/downtime-monitoring/notification-settings/index.tsx

```diff
@@ -11,20 +11,22 @@ import {
 	availableNotificationDurations as durations,
 	mobileAppLink,
 } from '../../sites-overview/utils';
-import type { MonitorSettings } from '../../sites-overview/types';
+import type { MonitorSettings, Site } from '../../sites-overview/types';
 
 import './style.scss';
 
 type Duration = { label: string; time: number };
 
 interface Props {
-	sites: Array< { blog_id: number; url: string } >;
+	sites: Array< Site >;
 	onClose: () => void;
 	settings?: MonitorSettings;
 }
 
 export default function NotificationSettings( { onClose, sites, settings }: Props ) {
 	const translate = useTranslate();
+
+	// Filter out the sites with monitor disabled
```

Comment:
Is this comment still relevant?

Reply:
No, it is not. I have removed it now - https://github.com/Automattic/wp-calypso/pull/71792/commits/b5979d78e9546d848d498216827889cd4f156a5d